user-guide/toc.yml

```diff
@@ -1492,6 +1492,11 @@
           topicUid: Features_of_a_DMS_Serial_Gateway
         - name: Using a DMS Serial Gateway
           topicUid: Using_a_DMS_Serial_Gateway
+    - name: Swarming
+      topicUid: Swarming
+      items:
+        - name: Switching objects
+          topicUid: SwitchingObjects
```

Comment:
I have moved the new Swarming section from 'DataMiner Modules' to 'Advanced DataMiner functionality for administrators' as we try to limit the former to discussing actual separate modules in Cube (e.g. Scheduler). Swarming seems to be more in line with e.g. [DCF](https://docs.dataminer.services/user-guide/Advanced_Functionality/DCF/About_the_DataMiner_Connectivity_Framework.html), which is also located underneath the advanced functionality section. Let me know if you disagree!